backend/backend-transit/src/main/java/eu/socialedge/hermes/backend/transit/domain/VehicleType.java

```diff
@@ -37,5 +41,17 @@
     /**
      * All other vehicles will return this type.
      */
-    OTHER
+    OTHER;
+
+    public static VehicleType fromNameOrOther(String name) {
+        if (isBlank(name))
+            return VehicleType.OTHER;
+
+        for (val type : values()) {
+            if (type.name().equalsIgnoreCase(name))
+                return type;
+        }
+
+        return VehicleType.OTHER;
```

Comment:
Could be eased with stream like
`return Arrays.stream(values()).filter(type->type.name().equalsIgnoreCase(name)).findFirst.orElse(VehicleType.OTHER);`